CHANELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 0.73.5 (2024-08-12)
```

Comment:
@kamieater1240 Let's merge these changelogs into a single version update.

Reply:
OK, changed back to the single version update.

Reply:
Thank you!